### PR TITLE
[DS] Typography in Users&Permissions/ Advanced settings

### PR DESCRIPTION
**Pull request by ronronscelestes**
## What

Replaced Text components with Typography in U&P / Advanced settings

### CODE REVIEW

File: packages/plugins/users-permissions/admin/src/pages/AdvancedSettings/index.js [hunk 1]
```diff
@@ -169,12 +169,12 @@ const AdvancedSettingsPage = () => {
                   paddingRight={7}
                 >
                   <Stack size={4}>
-                    <H3>
+                    <Typography variant="delta" as="h2">
```
Comment (HichamELBSI) [inline packages/plugins/users-permissions/admin/src/pages/AdvancedSettings/index.js:172]: Shouldn't it be a h3 ?

**Review by HichamELBSI — CHANGES_REQUESTED** [review]

File: packages/plugins/users-permissions/admin/src/pages/AdvancedSettings/index.js (hunk 1 above)
Comment (ronronscelestes) [reply packages/plugins/users-permissions/admin/src/pages/AdvancedSettings/index.js:172]: I'm not sure, the view only has an h1 (Advanced Settings title) so I thought Settings should be h2 - do you think h3 is better?

Screenshot 2021-11-15 at 18 29 13

Comment (HichamELBSI) [reply packages/plugins/users-permissions/admin/src/pages/AdvancedSettings/index.js:172]: Yes you are right ! Thanks 😄

**Review by HichamELBSI — APPROVED** [review]
LGTM

**Pull request merged**